indico/modules/logs/templates/entry_email.html

```diff
@@ -87,4 +101,52 @@
             {% endif %}
         </td>
     </tr>
+    <tr class="i-table content">
+        <td class="i-table caption log-caption">
+            {% trans %}Status{% endtrans %}
+        </td>
+        <td class="i-table value">
+            {% if data.state == 'pending' %}
+                {% trans %}Pending{% endtrans %}
```

Comment:
Let's add a bit more info for the translators (also for `Sent` and `Failed`):
```suggestion
                {# i18n: Status #}
                {% trans 'status' %}Pending{% endtrans %}
```

Reply:
do we need that first line for each status or can it go outside the if block?